### PR TITLE
Modification to create interest and assign  API to store the details of orgName

### PR DESCRIPTION
**Pull request by ruksana2808**

### CODE REVIEW

File: src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java [hunk 1]
```diff
@@ -71,11 +75,19 @@ public class InterestServiceImpl implements InterestService {
...
   @Override
   public CustomResponse createInterest(JsonNode interestDetails) {
     log.info("InterestServiceImpl::createInterest:entered the method: "+interestDetails);
     CustomResponse response = new CustomResponse();
     payloadValidation.validatePayload(Constants.INTEREST_VALIDATION_FILE_JSON,interestDetails);
+    Map<String, Object> propertyMap = new HashMap<>();
+    propertyMap.put(Constants.ID,interestDetails.get(Constants.ORG_ID).asText());
+    List<Map<String, Object>> orgDetails = cassandraOperation.getRecordsByPropertiesWithoutFiltering(
+        Constants.KEYSPACE_SUNBIRD, Constants.ORG_TABLE, propertyMap, null, 1);
+    String orgName = (String) orgDetails.get(0).get(Constants.USER_ROOT_ORG_NAME);
```
Comment (Sahil-tarento) [inline src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java:96]: check the null pointer expection for orgDetails page

File: src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java [hunk 2]
```diff
@@ -84,6 +96,7 @@ public CustomResponse createInterest(JsonNode interestDetails) {
       String interestId = String.valueOf(interestIdUuid);
       interest.setInterestId(interestId);
       ((ObjectNode) interestDetails).put(Constants.INTEREST_ID_RQST, String.valueOf(interestId));
+      ((ObjectNode) interestDetails).put(Constants.ORG_NAME, orgName);
```
Comment (Sahil-tarento) [inline src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java:105]: Why you do type casting on each object what is interest detail object

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java (hunk 2 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java:105]: InterestDetails is the jsonNode which is the payload, in the payload I need to put extra these fields to store in es so using these typecasting as directly we cannot put values to jsonNode

File: src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java (hunk 1 above)
Comment (ruksana2808) [reply src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java:96]: Modified

File: src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java [hunk 3]
```diff
@@ -71,11 +76,24 @@ public class InterestServiceImpl implements InterestService {
...
   @Override
   public CustomResponse createInterest(JsonNode interestDetails) {
     log.info("InterestServiceImpl::createInterest:entered the method: "+interestDetails);
     CustomResponse response = new CustomResponse();
     payloadValidation.validatePayload(Constants.INTEREST_VALIDATION_FILE_JSON,interestDetails);
+    Map<String, Object> propertyMap = new HashMap<>();
+    propertyMap.put(Constants.ID,interestDetails.get(Constants.ORG_ID).asText());
+    List<Map<String, Object>> orgDetails = cassandraOperation.getRecordsByPropertiesWithoutFiltering(
+        Constants.KEYSPACE_SUNBIRD, Constants.ORG_TABLE, propertyMap, null, 1);
+    if(ObjectUtils.isEmpty(orgDetails)){
```
Comment (Sahil-tarento) [inline src/main/java/com/igot/cb/interest/service/impl/InterestServiceImpl.java]: @ruksana2808 please do the proper formating and use collectionUtils as it a list

**Review by Sahil-tarento — CHANGES_REQUESTED** [review]

**Pull request merged**